protobufs/livekit_sip.proto

```diff
@@ -521,6 +521,7 @@ message DeleteSIPDispatchRuleRequest {
 message CreateSIPParticipantRequest {
   // What SIP Trunk should be used to dial the user
   string sip_trunk_id = 1;
+  SIPOutboundTrunkInfo sip_trunk = 20;
```

Comment:
IMO it seems rather confusing to embed *all* of the options from a SIP Trunk here. I would vote for just including the fields that are needed explicitly
- url
- username
- password